src/stores/CheckoutController.ts

```diff
@@ -39,9 +40,10 @@ export const createOrder = async (offerId: string, paymentMethodId?: number) =>
 };
 
 export const updateOrder = async (orderId: number, paymentMethodId?: number, couponCode?: string | null) => {
-  const {
-    config: { cleengId, cleengSandbox },
-  } = useConfigStore.getState();
+  const cleeng = useConfigStore.getState().config?.integrations?.cleeng;
```

Comment:
Would it make sense to make a helper method for this so we don't need to copy it and can still use desctructuring?

Reply:
You are right, I will create a getter for this one.